core/src/components/picker-column/picker-column.tsx

```diff
@@ -159,7 +178,7 @@ export class PickerColumnCmp implements ComponentInterface {
       }
 
       // Update transition duration
-      if (this.noAnimate) {
+      if (this.noAnimate || this.didColChange) {
```

Comment:
`this.update` accepts a `duration` parameter that lets us customize the duration of the animation. I think this code is going to be hard to maintain now that there are 3 different ways to skip the animation. Rather than check `didColChange` to skip the animation could we make use of the `duration` parameter instead?

You could update `refresh` to accept an `animated` parameter. From there, you'd update the `refresh` body to have the following code:

```js
this.update(y, animated === false ? 0 : TRANSITION_DURATION, true);
```

And then you'd update the `this.refresh` call in `onDomChange` to pass in `this.refresh(forceRefresh, false)` where `false` is the `animated` paramter.